sdk/cli/src/utils/config.ts

```diff
@@ -82,24 +82,20 @@ export async function getInstanceCredentials(
   env?: Partial<DotEnv>,
 ): Promise<{ personalAccessToken: string } | undefined> {
   const config = await readConfig();
-  note(`Config from readConfig is ${JSON.stringify(config, null, 2)}`, "debug", env?.SETTLEMINT_DEBUG);
-
   const instanceConfig = config.instances[instance];
-  note(
-    `Instance config is ${instanceConfig ? JSON.stringify(instanceConfig, null, 2) : instanceConfig}`,
-    "debug",
-    env?.SETTLEMINT_DEBUG,
-  );
 
   if (!instanceConfig) {
-    return undefined;
+    cancel(
```

Comment:
This would break the 'personalAccessTokenPrompt'. If this returns undefined it will show the prompt